npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -43,10 +45,32 @@ def fetch_files
         fetched_files += workspace_package_jsons
         fetched_files += lerna_packages
         fetched_files += path_dependencies(fetched_files)
+        instrument_package_manager_version
 
         fetched_files.uniq
       end
 
+      def instrument_package_manager_version
+        package_managers = {}
+
+        if [package_lock].any?
+          [package_lock].each do |lockfile|
+            versions = [package_managers["npm"]]
+            versions << Helpers.npm_version_numeric(lockfile.content)
+            package_managers["npm"] = versions.compact.min
+          end
+        end
+
```

Comment:
It looks like we'll only ever have 1 `package-lock.json` here so we can simplify this?

```suggestion
        package_managers["npm"] =  Helpers.npm_version_numeric(package_lock.content) if package_lock
```